packages/components/molecules/f-tabs/src/components/Tab.vue

```diff
@@ -63,8 +63,8 @@ export default {
 
         transitionName () {
             return this[TABS_COMPONENT].animationDirection === DIRECTION.LEFT
-                ? this.$style['fade-in-right']
```

Comment:
Similar thing to above comment, $style can not be picked up in jest when using `experimentalCSSCompile: false` so have updated the test.